libplacebo seems to fail to build here:
```console
error: builder for '/nix/store/9zfpczw05l9r22vbqayyq43j1lghnhjn-libplacebo-4.208.0.drv' failed with exit code 1;
       last 10 log lines:
       > [48/52] Compiling C object src/libplacebo.so.208.p/opengl_utils.c.o
       > [49/52] Compiling C object src/libplacebo.so.208.p/opengl_gpu_tex.c.o
       > [50/52] Compiling C object src/libplacebo.so.208.p/opengl_gpu_pass.c.o
       > [51/52] Compiling C++ object src/libplacebo.so.208.p/glsl_glslang.cc.o
       > FAILED: src/libplacebo.so.208.p/glsl_glslang.cc.o
       > g++ -Isrc/libplacebo.so.208.p -Isrc -I../src -Isrc/include -I../src/include -I/nix/store/862ndg8ph789sqyszqkdw2ginylf285x-libunwind-1.6.2-dev/include -I/nix/store/52sbwxkfdiyr0dvv0nanz618p6vkk1f6-lcms2-2.13.1-dev/include -I/nix/store/ph180wkgklhfj5wazjna16s8hmc8y4ns-shaderc-2022.4-dev/include -I/nix/store/bszqaw7ycydvqffylvr0va0van0hi0is-vulkan-headers-1.3.231.0/include -I/nix/store/6hxd0x3d1nkkyara7w8aq5sv3pj5ra8a-libepoxy-1.5.10-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Wextra -std=c++11 -Wundef -Wshadow -Wparentheses -Wpointer-arith -fvisibility=hidden -fPIC -pthread -MD -MQ src/libplacebo.so.208.p/glsl_glslang.cc.o -MF src/libplacebo.so.208.p/glsl_glslang.cc.o.d -o src/libplacebo.so.208.p/glsl_glslang.cc.o -c ../src/glsl/glslang.cc
       > ../src/glsl/glslang.cc:224:1: error: cannot convert '<brace-enclosed initializer list>' to 'int' in initialization
       >   224 | };
       >       | ^
       > ninja: build stopped: subcommand failed.
```

--- a/pkgs/development/compilers/shaderc/default.nix
+++ b/pkgs/development/compilers/shaderc/default.nix
@@ -8,25 +8,25 @@ let
   glslang = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "glslang";
-    rev = "18eef33bd7a4bf5ad8c69f99cb72022608cf6e73";
-    sha256 = "sha256-tkWVvYmSpJPaZ8VJOkAWndDWhA0PiHAkR3feBAo+knM=";
+    rev = "728c689574fba7e53305b475cd57f196c1a21226";
+    hash = "sha256-BAgDQosiO3e4yy2DpQ6SjrJNrHTUDSduHFRvzWvd4v0=";
   };
   spirv-tools = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "SPIRV-Tools";
-    rev = "c2d5375fa7cc87c93f692e7200d5d974283d4391";
-    sha256 = "sha256-tMJRljrlH+qb+27rTn+3LuEyMOVpiU0zSCiGNfUTb6g=";
+    rev = "d9446130d5165f7fafcb3599252a22e264c7d4bd";
+    hash = "sha256-fuYhzfkWXDm1icLHifc32XZCNQ6Dj5f5WJslT2JoMbc=";
   };
   spirv-headers = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "SPIRV-Headers";
-    rev = "0c28b6451d77774912e52949c1e57fa726edf113";
-    sha256 = "sha256-KpCMceTV/BRaoJe1qeXhKYQNQqdGaM6Q9nklpJKzHFY=";
+    rev = "c214f6f2d1a7253bb0e9f195c2dc5b0659dc99ef";
+    hash = "sha256-/9EDOiqN6ZzDhRKP/Kv8D/BT2Cs7G8wyzEsGATLpmrA=";
   };
 in
 stdenv.mkDerivation rec {
   pname = "shaderc";
-  version = "2021.0";
+  version = "2022.4";
 
   outputs = [ "out" "lib" "bin" "dev" "static" ];
 
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     owner = "google";
     repo = "shaderc";
     rev = "v${version}";
-    sha256 = "sha256-RfSMzrGVtdXbr/mjSrHoN447e3vMQfJbesQMvLOARBs=";
+    hash = "sha256-/p2gJ7Lnh8IfvwBwHPDtmfLJ8j+Rbv+Oxu9lxY6fxfk=";
   };
 
   patchPhase = ''

--- a/pkgs/development/libraries/libplacebo/default.nix
+++ b/pkgs/development/libraries/libplacebo/default.nix
@@ -7,7 +7,6 @@
 , vulkan-headers
 , vulkan-loader
 , shaderc
-, glslang
 , lcms2
 , libepoxy
 , libGL
@@ -38,7 +37,6 @@ stdenv.mkDerivation rec {
     vulkan-headers
     vulkan-loader
     shaderc
-    glslang
     lcms2
     libepoxy
     libGL
@@ -50,6 +48,7 @@ stdenv.mkDerivation rec {
     "-Dvulkan-registry=${vulkan-headers}/share/vulkan/registry/vk.xml"
     "-Ddemos=false" # Don't build and install the demo programs
     "-Dd3d11=disabled" # Disable the Direct3D 11 based renderer
+    "-Dglslang=disabled" # rely on shaderc for GLSL compilation instead
   ] ++ lib.optionals stdenv.isDarwin [
     "-Dunwind=disabled" # libplacebo doesn’t build with `darwin.libunwind`
   ];

